src/main/java/uk/gov/hmcts/reform/pip/data/management/service/artefact/ArtefactDeleteService.java

```diff
@@ -189,20 +171,24 @@ public String deleteAllArtefactsWithLocationNamePrefix(String prefix) {
         List<Artefact> artefactsToDelete = Collections.emptyList();
         if (!locationIds.isEmpty()) {
             artefactsToDelete = artefactRepository.findAllByLocationIdIn(locationIds);
-
-            if (!artefactsToDelete.isEmpty()) {
-                artefactsToDelete.forEach(a -> deleteAllPublicationBlobData(a));
-
-                List<UUID> artefactIds = artefactsToDelete.stream()
-                    .map(Artefact::getArtefactId)
-                    .toList();
-                artefactRepository.deleteAllByArtefactIdIn(artefactIds);
-            }
+            artefactsToDelete.forEach(this::handleArtifactDeletion);
         }
         return String.format("%s artefacts(s) deleted for location name starting with %s",
                              artefactsToDelete.size(), prefix);
     }
 
+    public void handleArtifactDeletion(Artefact artefact) {
```

Comment:
I think these two methods can be made private

Reply:
You are right they should be private. Thanks I have now changed it :)